source/support/support-overview.rst

```diff
@@ -0,0 +1,179 @@
+=======
+Support
+=======
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+Overview
+--------
+
+MinIO users have two options for support.
+
+#. Community support from the `public Slack channel <https://minio.slack.com>`_.
+#. A paid commercial license grants access to the MinIO Subscription Network, `SUBNET <https://subnet.min.io>`_.
```

Comment:
Might want to check in we/ marketing to see if they have a preferred lead in to describing SUBNET.

Spitballing:

MinIO `SUBNET <https://subnet.min.io?ref=docs>` providers 24/7 Direct-to-engineering support w/ 24 hour or 1 hour SLA depending on subscription level.